cdap-docs/developers-manual/source/data-exploration/custom-datasets.rst

```diff
@@ -2,10 +2,10 @@
     :author: Cask Data, Inc.
     :copyright: Copyright © 2014-2015 Cask Data, Inc.
 
-.. _data-exploration:
+.. _custom-dataset-exploration:
 
 ============================================
-Data Exploration
+Custom Dataset Exploration
```

Comment:
Got it. I see you have moved that to another file. Ignore this and the previous comment.